src/coreclr/vm/interpexec.cpp

```diff
@@ -1252,6 +1252,31 @@ void InterpExecMethod(InterpreterFrame *pInterpreterFrame, InterpMethodContextFr
 
                     goto CALL_INTERP_SLOT;
                 }
+                case INTOP_NEWOBJ_ARR:
```

Comment:
I would call the opcode `INTOP_NEW_MDARR` to make it clear that it allocates multi-dimensional arrays.

Reply:
Updated to INTOP_NEWMDARR to match INTOP_NEWARR.